crates/ruff/src/rules/flake8_todo/snapshots/ruff__rules__flake8_todo__tests__invalid-capitalization-in-todo_TDO006.py.snap

```diff
@@ -0,0 +1,38 @@
+---
+source: crates/ruff/src/rules/flake8_todo/mod.rs
+---
+TDO006.py:4:1: TDO006 [*] Invalid TODO capitalization: `ToDo` should be `TODO`
+  |
+4 | # TODO(evanrittenhouse): this is a valid TODO
+5 | # TDO006 - error
+6 | # ToDo(evanrittenhouse): invalid capitalization
+  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ TDO006
```

Comment:
Highlighting the whole comment can become very noisy for long comments. I think it would be better only to highlight the problematic section

```suggestion
6 | # ToDo(evanrittenhouse): invalid capitalization
  |      ^^^^  TDO006
```

This applies to all diagnostics in this PR.